src/connection/connection.js

```diff
@@ -75,7 +75,7 @@ export type ConnectionArguments = {
 
 type ConnectionConfig = {
   name?: string,
-  nodeType: GraphQLNamedType | GraphQLNonNull<GraphQLNamedType>,
+  nodeType: GraphQLOutputType,
```

Comment:
![image](https://user-images.githubusercontent.com/8336157/129717804-c736b394-b612-4036-9099-945307cdf0a2.png)

@saihaj Output types include arrays :( so we can't use it
How about you add `GraphQLInputNamedType` and `GraphQLOutputNamedType` to `graphql-js`?

Reply:
This was merged and released to `16.x.x` alpha channel https://github.com/graphql/graphql-js/pull/3063 as we discussed on Slack I will update to use alpha for `main` and then I can rebase this branch